presto-native-execution/presto_cpp/main/types/PrestoToVeloxConnector.cpp

```diff
@@ -1109,13 +1109,15 @@ HivePrestoToVeloxConnector::toVeloxSplit(
   for (const auto& [key, value] : hiveSplit->storage.serdeParameters) {
     serdeParameters[key] = value;
   }
-  std::unordered_map<std::string, std::string> infoColumns;
-  infoColumns.reserve(2);
-  infoColumns.insert(
-      {"$file_size", std::to_string(hiveSplit->fileSplit.fileSize)});
-  infoColumns.insert(
+  std::unordered_map<std::string, std::string> infoColumns = {
```

Comment:
We could follow this up with some code cleanup in Velox which uses constant strings for path and bucket synthesized columns. Is that the intention going forward ?

The info columns are used by Gluten as well. Might be worth checking with them.

Reply:
Yes that is why I add these.  Gluten can add any info columns they want in their split creation.